File: fix_jsx.py
with open('src/components/PersonDetailModal.tsx', 'r') as f:
    lines = f.readlines()

# Remove line 1053 (index 1052)
if "</div>" in lines[1052]:
    lines[1052] = ""

# Remove line 1035 (index 1034)
if "</div>" in lines[1034]:
    lines[1034] = ""

# Remove line 1002 (index 1001)
if "</div>" in lines[1001]:
    lines[1001] = ""

with open('src/components/PersonDetailModal.tsx', 'w') as f:
    f.writelines(lines)

File: fix_jsx.js
const fs = require('fs');
let code = fs.readFileSync('src/components/PersonDetailModal.tsx', 'utf8');

// A quick and dirty tag counter to find missing closing divs inside modalInner
const modalInnerMatch = code.match(/const modalInner = \([\s\S]*?\n\s*\);/);
if (modalInnerMatch) {
  const inner = modalInnerMatch[0];
  const divs = (inner.match(/<div/g) || []).length;
  const cdivs = (inner.match(/<\/div>/g) || []).length;
  console.log(`Open divs: ${divs}, Closed divs: ${cdivs}`);
}
